### ISSUE TITLE
Delete old database backups

### ISSUE THREAD

**Issue by REBELinBLUE** [REBELinBLUE/deployer#328]
The updater backs up the database before running, but it never deletes old backups. Should be changed to only keep a few backups

**Comment by bbashy** [comment]
Would this package not be better? https://github.com/spatie/laravel-backup

Can backup files as well as DB + clean them.

**Comment by REBELinBLUE** [comment]
It already uses that, just don't have it set up to purge old backups currently ;)

**Comment by REBELinBLUE** [comment]
Tell a lie, currently using https://github.com/backup-manager/laravel I will have a look at that package

**Comment by bbashy** [comment]
Yeah I checked to see what you used. Didn't see a purge/clean command for backup-manager?

**Comment by REBELinBLUE** [comment]
Yeah it doesn't seem to have it
